### PR TITLE
Add --all-tags test to pull

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

Adds a test for --all-tags to pull test suite.

### CODE REVIEW

File: tests/pull.bats [hunk 1]
```diff
@@ -184,3 +184,16 @@ load helpers
   [ "$status" -eq 0 ]
   [[ "$output" =~ "ostree-repo:latest" ]]
 }
+
+
+@test "pull-all-tags" {
+  run buildah pull --all-tags alpine
```
Comment (nalind) [inline tests/pull.bats]: Adding a `--signature-policy ${TESTSDIR}/policy.json` here should fix the CI errors.

Comment (TomSweeneyRedHat) [reply tests/pull.bats]: ugh, one day I'll remember to add them after I finish testing in my sandbox.  Thanks @nalind.

**Comment by TomSweeneyRedHat** [comment]
Repushed with the signature-policy added.

**Comment by nalind** [comment]
LGTM
@rh-atomic-bot r+

**Pull request closed without merging**